draft-ietf-ppm-dap.md

```diff
@@ -2731,90 +2611,65 @@ chosen independently of the generated reports.
-about individual participants. Aggregators must enforce the agreed-upon minimum
-batch size during the collect protocol, but implementations may also opt out of
+includes too few reports, then the aggregate result can reveal information about
+individual measurements. Aggregators enforce the agreed-upon minimum batch size
+during the collection protocol, but implementations SHOULD also opt out of
 participating in a DAP task if the minimum batch size is too small. This
-document does not specify how to choose minimum batch sizes.
+document does not specify how to choose an appropriate minimum batch size.
 
 ### VDAFs and compute requirements
 
-The choice of VDAF can impact the computation required for a DAP Task. For
+The choice of VDAF can impact the computation required for a DAP task. For
 instance, the Poplar1 VDAF {{!VDAF}} when configured to compute a set of heavy
 hitters requires each measurement to be of the same bit-length which all parties
 need to agree on prior to VDAF execution. The computation required for such
-tasks can increase superlinearly as multiple rounds of evaluation are needed for
-each bit of the measurement value.
+tasks increases superlinearly as a function of the chosen bit-length, as
+multiple rounds of evaluation are needed for each bit of the measurement value,
+and each bit of the measurement value requires computation which scales with the
+chosen bit-length.
 
 When dealing with variable length measurements (e.g domain names), it is
-necessary to pad them to convert into fixed-size measurements. When computing
-the heavy hitters from a batch of such measurements, we can early-abort the
-Poplar1 execution once we have reached the padding region for a candidate
+necessary to pad them to convert into fixed-length measurements. When computing
+the heavy hitters from a batch of such measurements, VDAF execution can be
+terminated early once once the padding region is reached for a given
 measurement. For smaller length measurements, this significantly reduces the
 cost of communication between Aggregators and the steps required for the
 computation. However, malicious Clients can still generate maximum length
 measurements forcing the system to always operate at worst-case performance.
 
-[[TODO: Revisit this paragraph once https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/273
-is resolved.]]
+[[TODO: Revisit this paragraph once
+https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/273 is resolved.]]
 
-Therefore, care must be taken that a DAP deployment can comfortably handle
-computation of measurements for arbitrarily large sizes, otherwise, it may
-result in a DoS possibility for the entire system.
+Therefore, care must be taken that a DAP deployment can handle VDAF execution of
+all possible measurement values for any tasks that the deployment is configured
+for. Otherwise, an attacker may deny service by uploading many expensive
+reports.
```

Comment:
Hmm -- while availability/denial-of-service is often considered a security concern, I agree this section doesn't fit in well since all the other sections are more related to the robustness/privacy properties. Moved.

There was already a subsection ("Data Resolution Limitations") in the Operational Considerations section which touches on basically the same ideas; I pulled a little bit of information in from that subsection & eliminated it.